test/parallel/test-stream-manualstart.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+fs.promises.open(__filename).then(common.mustCall((fd) => {
+  const rs = new fs.ReadStream(null, {
+    fd: fd,
+    manualStart: false
+  });
+  setTimeout(() => assert(rs.bytesRead > 0), common.platformTimeout(10));
```

Comment:
this should be a setImmediate.

Reply:
The read handler is already in a `setImmediate` - so this will run the assert before Node has a chance to start reading data.
I don't have any ideas on how to achieve this without a timer.

Reply:
thanks for the clarification. Maybe add this in a comment?